packages/broker/src/ConfigWizard.ts

```diff
@@ -1,38 +1,46 @@
 import inquirer from 'inquirer'
 import { Wallet } from 'ethers'
-import { writeFileSync, mkdirSync, existsSync } from 'fs'
+import { Config } from './config'
 import path from 'path'
+import { writeFileSync, existsSync, mkdirSync } from 'fs'
+import * as os from 'os'
+import chalk from "chalk"
 
-import { Config } from './config'
+function logger(...args: any[]) {
+    console.log(chalk.bgWhite.black(':'), ...args)
+}
 
-import * as os from 'os'
+logger.info = (...args: any[]) => {
+    console.log(chalk.bgWhite.black(':', ...args))
+}
 
-const logger = {
-    log: (...args: any[]) => {
-        console.log('\x1b[7m' + ':' +'\x1b[0m', ...args)
-    },
-    info: (...args: any[]) => {
-        console.log('\x1b[47m' +  '\x1b[30m' + ':', ...args, '\x1b[0m')
-    },
-    warn: (...args: any[]) => {
-        console.log('\x1b[33m' + '!' + '\x1b[0m', ...args)
-    },
-    error: (...args: any[]) => {
-        console.log('\x1b[31m' + '!' + '\x1b[0m', ...args)
-    },
-    alert:(...args: any[]) => {
-        console.log('\x1b[43m' + '\x1b[30m' + '!', ...args, '\x1b[0m')
-    }
+logger.alert = (...args: any[]) => {
+    console.log(chalk.bgYellow.black('!', ...args))
+}
+
+logger.warn = (...args: any[]) => {
+    console.log(chalk.bgYellow.black('!'), ...args)
+}
+
+logger.error = (...args: any[]) => {
+    console.log(chalk.bgRed.black('!'), ...args)
 }
 
-const DEFAULT_WS_PORT = 7170
-const DEFAULT_MQTT_PORT = 7171
-const DEFAULT_HTTP_PORT = 7172
-const DEFAULT_LEGACY_WS_PORT = 7173
+const MIN_PORT_VALUE = 1024
+const MAX_PORT_VALUE = 49151
 
-const DefaultConfig: Config = {
-    ethereumPrivateKey: '',
-    generateSessionId: false,
+import * as WebsocketConfigSchema from './plugins/websocket/config.schema.json'
+import * as MqttConfigSchema from './plugins/mqtt/config.schema.json'
+//import * as HttpConfigSchema from './plugins/publishHttp/config.schema.json'
+import * as LegacyWebsocketConfigSchema from './plugins/legacyWebsocket/config.schema.json'
+
+
+const DEFAULT_WS_PORT = WebsocketConfigSchema.properties.port.default
+const DEFAULT_MQTT_PORT = MqttConfigSchema.properties.port.default
+const DEFAULT_HTTP_PORT = 8585//HttpConfigSchema.properties.port.default
```

Comment:
Why this is not read from configs? And the value is different from the Broker's default `httpServer` port?

Reply:
this is the one we were commenting on, the path is not the indicated since the plugin shouldn't have a default value for it.

Reply:
But currently it is used in `publishHttp: { port: DEFAULT_HTTP_PORT }`. That http port question should be written into the root level httpServer block if user wants to change the default. And if user doesn't want to change, we shouldn't write any port value